src/TestHarness/Main.Designer.cs

```diff
@@ -39,8 +39,8 @@ private void InitializeComponent()
             // 
             // Devices
             // 
-            this.Devices.Anchor = ((System.Windows.Forms.AnchorStyles)(((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Left)
-                        | System.Windows.Forms.AnchorStyles.Right)));
+            this.Devices.Anchor = ((System.Windows.Forms.AnchorStyles)(((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Left) 
+            | System.Windows.Forms.AnchorStyles.Right)));

```

Comment:
We probably didn't have the right amount of whitespace here before, but I think it reads better with at least a space or two to indicate that it's a continuation of the above line. It also seems odd to me that we separate the Top, Left, and Right flags with parentheses like that (i.e. it looks like this was odd before, and I just never noticed until now).

Reply:
(This file is autogenerated as mentioned in the comment below)